src/main/java/com/rackspace/salus/common/web/URLDecodeFilter.java

```diff
@@ -26,9 +27,8 @@
 /**
  * A filter to make sure that our inputs are properly URLdecode'ed
  *
- * To apply this to a particular project a configuration needs to be written
- * to apply this to specific URL patterns. It is likely ok to apply this to
- * every URI in a project.
+ * To apply this to a project add @EnableURLDecode to the main function that starts spring boot
```

Comment:
`@` symbols are special in Javadoc, so replacing with a link is an easy solution:
```suggestion
 * To apply this to a project add {@link EnableURLDecode} to the main function that starts spring boot
```